src/Control/Carrier/Class.hs

```diff
@@ -43,3 +57,80 @@ instance Monoid w => Carrier (Writer w) ((,) w) where
   eff (Tell w (w', k))    = (mappend w w', k)
   eff (Listen m k)        = uncurry k m
   eff (Censor f (w, a) k) = let (w', a') = k a in (mappend (f w) w', a')
+
+
+-- transformers
+
+instance (Carrier sig m, Effect sig) => Carrier (Error e :+: sig) (Except.ExceptT e m) where
+  eff (L (Throw e))     = Except.throwE e
+  eff (L (Catch m h k)) = Except.catchE m h >>= k
```

Comment:
I’d been writing these all out long-form for a while when it hit me that I can do it so much more easily with the operations `transformers` exports.

Reply:
This is remarkable to me for many reasons, but most notably because it’s a visceral display of `fused-effects` as an algebraic interface to `mtl`.

Reply:
> This is remarkable to me for many reasons, but most notably because it’s a visceral display of `fused-effects` as an algebraic interface to `mtl`.

…algebraic interface to _`transformers`_.